hack/run-latency-testing.sh

```diff
@@ -15,6 +16,12 @@ if ! which tput &> /dev/null 2>&1 || [[ $(tput -T$TERM colors) -lt 8 ]]; then
   NO_COLOR="-noColor"
 fi
 
+if [[ -z "${CNF_TEST_IMAGE}" ]]; then
```

Comment:
Same comment as above:



I see 2 occurrences of the env var:

    CNF_TEST_IMAGE
    CNF_TESTS_IMAGE

    Which one is the right one ?

Reply:
CNF_TESTS_IMAGE is the right one